docker/ckb-indexer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM nervos/ckb-docker-builder:bionic-rust-1.45.2 as ckb-indexer-builder
+
+WORKDIR /var/lib
+
+RUN git clone https://github.com/nervosnetwork/ckb-indexer.git
+
+RUN cd ckb-indexer && git checkout 76f537e02250e3e6c4091c966bb7735f28e8974e && cargo build
+
+FROM ubuntu:bionic
+
+COPY --from=ckb-indexer-builder \
+     /usr/lib/x86_64-linux-gnu/libssl.so.* \
+     /usr/lib/x86_64-linux-gnu/libcrypto.so.* \
+     /usr/lib/x86_64-linux-gnu/
+
+COPY --from=ckb-indexer-builder \
+    /var/lib/ckb-indexer/target/debug/ckb-indexer /bin/
+
+EXPOSE 8116
+
+CMD RUST_LOG=info ckb-indexer -s /tmp/ckb-indexer-test -c http://ckb:8114
```

Comment:
`-c http://ckb:8114` only works when this container links to a ckb container named `ckb`.
Your can use a env var here. When we use ckb-indexer standalone, we can pass new env to the container.

ref: <https://vsupalov.com/docker-env-vars/>

Reply:
I can just remove `-c http://ckb:8114`, which is the default start command.

If someone wants customized start command, he can overwrite `CMD` with `docker run`